src/main/psxsdk/libgpu/sys.c

```diff
@@ -337,8 +365,25 @@ INCLUDE_ASM("main/nonmatchings/psxsdk/libgpu/sys", _reset);
 
 INCLUDE_ASM("main/nonmatchings/psxsdk/libgpu/sys", _sync);
 
-INCLUDE_ASM("main/nonmatchings/psxsdk/libgpu/sys", set_alarm);
+void set_alarm(void) {
+    D_80039254 = VSync(-1) + 240;
+    D_80039258 = 0;
+}
 
 INCLUDE_ASM("main/nonmatchings/psxsdk/libgpu/sys", get_alarm);
 
-INCLUDE_ASM("main/nonmatchings/psxsdk/libgpu/sys", GPU_memset);
+void GPU_memset(s8* arg0, s8 arg1, s32 arg2) {
```

Comment:
More compact version of it, I also borrowed the signature from `memset`: https://decomp.me/scratch/7yOnQ